iaso/models/openhexa.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+from iaso.utils.models.encrypted_text_field import EncryptedTextField
+from iaso.utils.models.soft_deletable import (
+    DefaultSoftDeletableManager,
+    IncludeDeletedSoftDeletableManager,
+    OnlyDeletedSoftDeletableManager,
+    SoftDeletableModel,
+)
+
+
+class OpenHEXAInstance(models.Model):
+    class Meta:
+        verbose_name = "OpenHEXA Instance"
+        verbose_name_plural = "OpenHEXA Instances"
+
+    name = models.CharField(max_length=255, unique=True)
+    url = models.URLField()
+    token = EncryptedTextField()
+    description = models.TextField(null=True, blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return "%s %s" % (self.name, self.url)
+
+
+class OpenHEXAWorkspace(SoftDeletableModel):
+    class Meta:
+        indexes = [
+            models.Index(fields=["slug"]),
+        ]
+        unique_together = ("openhexa_instance", "account", "slug")
+        verbose_name = "OpenHEXA Workspace"
+        verbose_name_plural = "OpenHEXA Workspaces"
+
+    openhexa_instance = models.ForeignKey(
+        OpenHEXAInstance, on_delete=models.CASCADE, related_name="openhexa_workspaces"
+    )
+    account = models.ForeignKey("iaso.Account", on_delete=models.CASCADE)
+    slug = models.CharField(max_length=255)
```

Comment:
```suggestion
    slug = models.SlugField(max_length=255)
```

https://docs.djangoproject.com/en/5.2/ref/models/fields/#slugfield

But CharField works just fine

Reply:
Interesting, didn't know about that one, I'll use it because it comes with some validation :+1: